tensorflow/tools/ci_build/install/install_pip_packages.sh

```diff
@@ -95,15 +95,15 @@ pip3 install --upgrade gast
 pip3 install --upgrade termcolor
 
 # Keras
-pip3 install keras-nightly --no-deps
+pip3 install keras-nightly==2.6.0.dev --no-deps
```

Comment:
This will not work as it will not find any exact version. Use the same string as in the mentioned forum line.